docs/openapi/components/schemas/credentials/GS18PrefixLicenseCredential.yml

```diff
@@ -69,14 +69,27 @@ properties:
       id:
         description: The DID for the GS1 Member Organization, or GS1 Global Office itself, to which the GS1-8 Prefix has been licensed.
         type: string
+      partyGLN:
+        description: A Global Location Number (GLN) is the GS1 Identification Key used to identify physical locations or parties. The key comprises a GS1 Company Prefix, Location Reference and Check Digit. For more information see https://www.gs1.org/gln.
+        type: string
+      organizationName:
+        description: The name of the organization expressed in text.
+        type: string
       licenceValue:
         description: The GS1-8 Prefix.
         type: string
+      alternativeLicenceValue:
+        description: Alternative value of the licence. For most licences, the alternative value of the licence is the bridge between the U.P.C. system and the rest of the GS1 system; if the licence value starts with zero, then the alternative licence value is the same as the licence value but without the starting zero.
+        type: string
+    required:
+      - partyGLN
+      - organizationName
+      - licenceValue
```

Comment:
GS1 standards are typically written in British english with the spelling licence used to convey the noun form.   We have deviated from this for the schema.org GS1 web vocabulary extension.  

I'll get an update for how we can change this so that the vocabulary terms match US english conventions in our next version of the data model.   But this may take a bit of time as I want to ensure it matches the published information.  Can we open an Issue to correct this and continue with this PR?

Reply:
I won't lie down on the road to block it. The terms might be kept British, with the definitions American ... or disclaimers added to description(s) explaining what may otherwise look like misspellings (in either/both direction(s))... I18n is hard. :-/

Reply:
Thanks for your assistance pushing past this guys. Here's an issue for us to come back to this: https://github.com/w3c-ccg/traceability-vocab/issues/705.

> I18n is hard
This is nothing... Just you wait until I start in Danish, throwing æ's ø's and å's around... :D